plugins/dell/fu-plugin-dell.c

```diff
@@ -621,6 +627,7 @@ fu_plugin_dell_detect_tpm (FuPlugin *plugin, GError **error)
 		fu_device_set_id (dev_alt, tpm_id_alt);
 		fu_device_add_guid (dev_alt, tpm_guid_alt);
 		fu_device_set_name (dev_alt, pretty_tpm_name_alt);
+		fu_device_set_summary (dev_alt, "Secondary TPM device");
```

Comment:
"Alternate operating mode for platform TPM device"

Reply:
Done